src/app/component/ProductGallery/ProductGallery.component.js

```diff
@@ -141,11 +182,11 @@ export default class ProductGallery extends PureComponent {
 
         switch (media_type) {
         case IMAGE_TYPE:
-            return this._renderImageItem(media, index);
+            return this.renderImage(media, index);
         case VIDEO_TYPE:
-            return this._renderVideoItem(media, index);
+            return this.renderVideo(media, index);
         case PLACEHOLDER_TYPE:
```

Comment:
I think, we can get rid of this, by which minimizing the amount of updates needed to initialize the slider. Maybe we could use the same render IMAGE_TYPE but there try to render **the same element structure** but with image src set to empty string. It will look like placeholder, but won't re-render when loaded => won't blink.

Reply:
Agreed to leave as is for now.